pom.xml

```diff
@@ -29,49 +29,51 @@
 	</developers>
 
 	<properties>
-		<java.version>11</java.version>
-		<swagger.version>2.9.2</swagger.version>
+		<java.version>21</java.version>
 		 <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-      	<maven.compiler.source>11</maven.compiler.source>
-		<maven.compiler.target>11</maven.compiler.target>
+      	<maven.compiler.source>21</maven.compiler.source>
+		<maven.compiler.target>21</maven.compiler.target>
 		<maven.compiler.version>3.8.0</maven.compiler.version>
 		<maven.surefire.plugin.version>2.22.0</maven.surefire.plugin.version>
 		<maven.jar.plugin.version>3.0.2</maven.jar.plugin.version>
 		<maven.war.plugin.version>3.1.0</maven.war.plugin.version>		
-		<maven.jacoco.version>0.8.5</maven.jacoco.version>
+		<maven.jacoco.version>0.8.11</maven.jacoco.version>
 		<maven.sonar.plugin.version>3.7.0.1746</maven.sonar.plugin.version>
-		<maven.javadoc.version>3.0.1</maven.javadoc.version>
-        <spring.boot.version>2.0.2.RELEASE</spring.boot.version>
-        <spring.data.jpa.version>2.0.7.RELEASE</spring.data.jpa.version>
+		<maven.javadoc.version>3.2.0</maven.javadoc.version>
         <spring.security.test.version>5.0.5.RELEASE</spring.security.test.version>
-        <spring-cloud-config.version>2.0.0.RELEASE</spring-cloud-config.version>
         <h2.version>1.4.197</h2.version>
         <jackson.datatype.version>2.9.8</jackson.datatype.version>
-        <swagger.version>2.9.2</swagger.version>
         <powermock.beta.version>2.0.7</powermock.beta.version>
-		<mockito.version>2.28.2</mockito.version>
         <sonar.coverage.exclusions>**/constant/**,**/config/**,**/cache/**,**/dto/**,**/model/**,**/exception/**,**/repository/**,**/security/**,**/*Config.java,**/*Application.java,**/*Handler.java</sonar.coverage.exclusions>
         <sonar.cpd.exclusions>**/dto/**,**/config/**,**/api/**</sonar.cpd.exclusions>
         <io.micrometer.prometheus.version>1.4.2</io.micrometer.prometheus.version>
         <gson.version>2.8.4</gson.version>
-        <kernel.websub.version>1.2.1-SNAPSHOT</kernel.websub.version>
+        <kernel.websub.version>1.2.1-java21-SNAPSHOT</kernel.websub.version>
```

Comment:
whether it is migrated then change to 1.2.1-SNAPSHOT

Reply:
updated